packages/components/psammead-navigation/src/index.stories.jsx

```diff
@@ -146,6 +147,16 @@ const navigationStory = (
     ? AmpScrollableNavigation
     : CanonicalScrollableNavigation;
 
+  const [isScrollable, setIsScrollable] = useState(false);
```

Comment:
Is there any difference setting it by default as `null` or `false`? Just asking because in the hooks we had `null`.

Reply:
The only place it gets used is in `isScrollable && ...` statements, so using `false` will lead to the same results as `null`.